graphos/templates/graphos/highcharts/js.html

```diff
@@ -2,6 +2,9 @@
 $(function () {
     $('#{{ chart.get_html_id }}').highcharts({
         chart: {{chart.get_chart_json | safe}},
+        exporting: {{chart.get_exporting_json | safe}},
```

Comment:
@yashrastogi16 You need to add these to map js and any other template used by highcharts.